src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -625,7 +625,7 @@ private void LogErrorOutput(string output)
 
         internal static void LogErrorOutput(string output, TaskLoggingHelper log)
         {
-            string[] lines = output.Split(new string[] { "\r\n" }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Can we just extract this into a protected readonly static and reference from both subclasses?

Reply:
Never mind -- if we have to take this escrow, please keep the change as is.

Reply:
whoops, force-pushing back to where it was